### ISSUE TITLE
Lesson 3 Task 2: Not recognizing correct answer

### ISSUE THREAD

**Issue by CarnegieL** [JetBrains/pycharm-courses#14]
Hi,

I'm unsure if I'm just being stupid, however for String multiplication, I've put in the answer:

`hello = "hello"
ten_of_hellos = hello * 10
print(ten_of_hellos)`

However, it's telling me to use "Multiplication" and marking it wrong... Am I missing something here?

**Comment by lancelote** [comment]
Hello @CarnegieL 
Nope, you are right. It's a known bug with a pending pull request (#11). You can fix it temporary by yourself - edit function `test_value()` inside the file `~/PycharmProjects/PythonIntroduction/lesson3/task2/tests.py` (placed next to the task file):

```python
def test_value():
    file = import_task_file()
    if hasattr(file, "ten_of_hellos") and file.ten_of_hellos == "hellohellohellohellohellohellohellohellohellohello":
        passed()
    else:
        failed("Use multiplication")
```

This will fix the broken test.

**Comment by alugarius** [comment]
i have the same problem, please fix that!
i want to use that in school

**Comment by veganaize** [comment]
Why isn't this fixed?  JetBrains is only hurting their chance for a potentially massive adoption of this IDE.

**Comment by veganaize** [comment]
REASON:  Windows version fails to remove X:\Users\<user-name>\PycharmProjects\<...>

So, even after complete uninstall & reinstall, the buggy course projects remain.  This is EXTRA frustrating for non-power users who wish to reset Pycharm's progress.  Anyway.

**Comment by medvector** [comment]
Hi @veganaize What PyCharm EDU version are you talking about? 
This bug was fixed in version 3.5.

**Comment by veganaize** [comment]
@medvector I am talking about Pycharm EDU 3.5 (current stable).  It fails to remove remnants of previous version(s) in user's home folder (on Windows 8.1 64-bit) -- Even when the two checkboxes, which remove those kinds of things, are checked.  Then, when I install 3.5 (after reboot & everything), it still loads the _OLD_ course lessons (v3.0) and my progress still remains at whatever it was (not reset).

It took me a lot of mucking to finally figure out (the obvious?) that I needed to completely remove/archive the **C:\Users\<username>\PycharmProjects\** folder & subfolders.

1. These folders are located in a weird place for Windows users, as they don't typically look in the folder named after their username (they usually look in subfolders such as "Downloads", "Documents", etc.).

2. It is counter-intuitive to offer an option to delete all caches, etc. when uninstalling -- but then leave those particular files laying around (without even a popup to offer to delete them).

3. It is counter-intuitive that the new version (v3.5) doesn't offer to "fix" (ie. overwrite) the existing corrupt lesson(s) (which should have been deleted already anyway; but since they weren't, it's now 3.x's or 4.x's job to do so).

_Thanks!_
